addons/l10n_it_edi/__manifest__.py

```diff
@@ -17,10 +17,10 @@
     'website': 'http://www.odoo.com/',
     'data': [
         'security/ir.model.access.csv',
-        'data/invoice_it_template.xml',
+        'data/account_withholding_report_data.xml',
         'data/invoice_it_simplified_template.xml',
+        'data/invoice_it_template.xml',
         'data/ir_cron.xml',
-        'data/account.account.tag.csv',
```

Comment:
Looks like it shouldn't be removed?

Reply:
No, there were 2 files for whatever reason.
The tags should not be part of the template anyway because they are not related to the company so it is just regular data.